### PR TITLE
Is709/db migration

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

Since we have (for the moment) a sole database for the framework, we have
- centralized models (i.e. chemas) as a package in [packages/postgres-database](packages/postgres-database): ``simcore-postgres-database==0.1.0``
- Added migration scripts
- New maintenance service based on a jupyter notebook (added in separate stack for convenience)

**TODOS**
-  ~[simcore-sdk/models](packages/simcore-sdk/src/simcore_sdk/models/DEPRECATED) was marked as deprecated. Check with @mguidon !~ -> Wait for @mguidon PR
- [x] All services are updated to new dependency
- [x] Includes changes in storage table
- [x] CLI to run migration (shall be installed with ``postgres-database``)

## Related issue number
Implements #709 

## Checklist

- [x] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [x] **Runs in the swarm**
- [ ] If you design a new module, add your user to .github/CODEOWNERS

### CODE REVIEW

File: services/maintenance/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#FROM nbgallery/jupyter-alpine:7.8.4
+FROM jupyter/base-notebook:python-3.7.3 AS base
+
+
+# COPY --chown=jovyan:users scripts/docker/healthcheck_curl_host.py /healthcheck/healthcheck_curl_host.py
```
Comment (sanderegg) [inline services/maintenance/Dockerfile]: you should use the same healthcheck as the one used with the jupyter notebook service.
I think it is important to know if the service really works or is responsive.

File: services/maintenance/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+RUN unset JUPYTERHUB_API_TOKEN
+
+COPY --chown=jovyan:users packages/postgres-database $HOME/postgres-database
+RUN pip install --upgrade pip wheel setuptools && \
+    pip install $HOME/postgres-database[migration] && \
+    rm -rf $HOME/postgres-database
+
+WORKDIR $HOME/work
+COPY --chown=jovyan:users services/maintenance/notebooks .
+RUN jupyter trust db-migration.ipynb
```
Comment (sanderegg) [inline services/maintenance/Dockerfile]: you can trust the whole folder so you do not need to add each file separately. see jupyter notebook service.

File: services/maintenance/boot.sh [hunk 3]
```diff
@@ -0,0 +1 @@
+exec start-notebook.sh --NotebookApp.password=$(python -c "from notebook.auth import passwd; print(passwd('${MAINTENANCE_PASSWORD}'))")
```
Comment (sanderegg) [inline services/maintenance/boot.sh]: nice!

File: services/sidecar/src/sidecar/_deprecated.py [hunk 4]
```diff
@@ -95,8 +95,11 @@ def _process_task_input(self, port, input_ports):
                 other_task = None
                 _session = self._db.Session()
                 try:
-                    other_task =_session.query(ComputationalTask).filter(and_(ComputationalTask.node_id==other_node_id,
-                                    ComputationalTask.project_id==self._task.project_id)).one()
+                    # pylint: disable=no-member
+                    other_task =_session.query(ComputationalTask).filter(
```
Comment (sanderegg) [inline services/sidecar/src/sidecar/_deprecated.py]: so this guy also uses the old ways... are you keeping the old one finally?

Comment (pcrespov) [reply services/sidecar/src/sidecar/_deprecated.py]: I talked to @mguidon and we will keep it like that for the moment. Will be slowly removed

**Pull request merged**